Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -65,8 +65,8 @@ For more details on the mons and when to choose a number other than `3`, see the
   * `disable`: is set to `true`, the crash collector will not run on any node where a Ceph daemon runs
   * `daysToRetain`: specifies the number of days to keep crash entries in the Ceph cluster. By default the entries are kept indefinitely.
 * `logCollector`: The settings for log collector daemon.
-  * `enabled`: if set to `true`, the log collector will run as a side-car next to each Ceph daemon. The Ceph configuration option `log_to_file` will be turned on, meaning Ceph daemons will log on files in addition to still logging to container's stdout. These logs will be rotated. (default: false)
-  * `periodicity`: how often to rotate daemon's log. (default: 24h). Specified with a time suffix which may be 'h' for hours or 'd' for days. **Rotating too often will slightly impact the daemon's performance since the signal briefly interrupts the program.**
+  * `enabled`: if set to `true`, the log collector will run as a side-car next to each Ceph daemon. The Ceph configuration option `log_to_file` will be turned on, meaning Ceph daemons will log on files in addition to still logging to container's stdout. These logs will be rotated. The coredump files will be generated in `/var/lib/systemd/coredump` directory on the host where the pod is running in case a daemon terminates with a segfault. (default: `true`)
```

Comment:
Are we sure this is happening for all underlying OSes/kernels? What happens when the `kernel.core_pattern` is not pointing to the `coredumpctl`/ just a path/directory.

Reply:
@galexrt @travisn please let me know if the new wording sounds good based on our discussion or we can make it better ?